src/app/pages/IdxPage/index.jsx

```diff
@@ -1,22 +1,28 @@
-import React, { Fragment } from 'react';
+import React, { Fragment, useContext } from 'react';
 import path from 'ramda/src/path';
-import PageContainer from '#lib/pageStyles/PageContainer';
+import IndexHeading from '#containers/IndexHeading';
+import IndexPageContainer from '#app/components/PageLayout/IndexPageContainer';
 import FrontPageSection from '#containers/FrontPageSection';
 import idxPageDataPropTypes from '#models/propTypes/idxPage';
+import { ServiceContext } from '#contexts/ServiceContext';
 
 const IdxPage = ({ pageData }) => {
+  const { script, service, dir } = useContext(ServiceContext);
   const groups = path(['content', 'groups'], pageData);
+  const heading = path(['metadata', 'title'], pageData);
 
   return (
     <main role="main">
-      <h1 id="content">IDX Page</h1>
-      <PageContainer>
+      <IndexPageContainer>
+        <IndexHeading id="content" script={script} service={service} dir={dir}>
```

Comment:
I think we can move the serviceContext props to the container level. This will mean you'll have to update quite a bit... 👁️ 👁️ 

(i.e to use ServiceContextProvider in storybook and test)

Reply:
I've done it like that before, but th

> Looks good to me overall, just a few comments.
> 
> I can see the heading added here: https://5d28eb3fe163f6002046d6fa-ivxhgahytc.chromatic.com/?path=/story/pages-idx-page--ukrainian-ukraine-in-russian
> 
> Also, do we have an issue capture:
> 
>     * dev-a11y for `psammead-heading-index`
> 
>     * using `src/app/containers/IndexHeading/index.jsx` in MostRead Page

They are in my to-do list 👍🏻

Reply:
I've initially done it like you suggest https://github.com/bbc/simorgh/pull/6673/commits/0712ff8112b8dfbbc6e824e927bfa106262b6288 but changed to how it is now to be able to pass the props from the story, however, I didn't think about using the `ServiceContext` also in the story 👍